apps/src/gamelab/dropletConfig.js

```diff
@@ -13,11 +14,23 @@ var COLOR_WHITE = '#FFFFFF';
 var COLOR_BLUE = '#64B5F6';
 var COLOR_ORANGE = '#FFB74D';
 
+var GameLab;
+
+exports.injectGameLab = function (gamelab) {
+  GameLab = gamelab;
```

Comment:
This made me cringe a little at first, but I see that we're doing it to conform to the config format droplet needs, so that's fine.  Nit: Can we name the global `gameLab` or `gamelab` instead of `GameLab` since it's an instance, rather than a reference to the constructor?